plugin/build.gradle.kts

```diff
@@ -44,7 +44,7 @@ configurations["functionalTestRuntimeOnly"].extendsFrom(configurations["testRunt
 functionalTest {
     testingStrategies.set(
         listOf(
-            strategies.coverageForGradleVersion("7.6"),
+            strategies.coverageForGradleVersion("8.4"),
```

Comment:
note: This is identical to `strategies.coverageForGradleVersion(GradleVersion.current().version)` at the moment. We probably need to remove the latter in this case.
Or we can update the gradle version in `gradle-wrapper.properties` to 8.7 as it is more stable than 8.8. In this case, we can do nothing with `strategies.coverageForGradleVersion(GradleVersion.current().version)`